guiprep.pl

```diff
@@ -1780,17 +1640,15 @@ sub viewersel{
 	-width =>		'14'
 )->pack(-side => 'left', -pady => '4', -padx => '10', -anchor => 'n');
 
-my $p8f3 = $page8->Frame()->pack(-side => 'top', -anchor => 'n',-expand =>'both', -fill => 'both');
+my $p8f3 = $page8->ttkFrame()->pack(-side => 'top', -anchor => 'n',-expand =>'1', -fill => 'both');
 
 my $displaybox = $p8f3->Scrolled('TextUndo',
 	-scrollbars => 'osoe',
-	-background => 'white',
 	-width => 	'100',
-	-font => $courl,
 	-height => 	'30',
 	-wrap => 	'word',
-)->pack(-side => 'top', -anchor => 'n', -padx => '6',-pady => '6', -expand =>'both', -fill => 'both');
-$displaybox->tagConfigure('highlight', background => 'yellow');
+)->pack(-side => 'top', -anchor => 'n', -padx => '6',-pady => '6', -expand =>'1', -fill => 'both');
+#$displaybox->tagConfigure('highlight', background => 'yellow'); #background => 'yellow'
```

Comment:
Left-in comment.

Reply:
I couldn't remember at first what it was, but now I do -- looks like background isn't currently implemented. I left it as a comment since that's still a "need to address somehow". Not sure what would be preferred instead.

Reply:
Then perhaps just a comment like:
```
# pre-pTk, background was set to 'yellow' but pTk doesn't support that
```

Reply:
Fixed with ad76a9b.